addons/compat_sog/CfgMagazines/belts.hpp

```diff
@@ -12,3 +12,6 @@ class vn_m16_mag_base;
 class vn_m63a_100_mag: vn_m16_mag_base {
     ACE_isBelt = 1;
 };
+class vn_mg42_50_mag: vn_lmgmag_base {
```

Comment:
I'm not a fan of this. Can MG42 drums be linked?

Reply:
If I understand it correctly, [Wikipedia](https://en.wikipedia.org/wiki/MG_42#:~:text=Feeding,-Method%20of%20joining&text=The%20MG%2042%20could%20use,housed%20in%20the%20feed%20cover.) says they cannot. Only through additional connectors, possibly.

Of course, it's not a standard practice in that case. Most of the time they would be loaded with 50-round drum magazines on the go.

Reply:
They can, but only in 50 round segments (ei. a 200 round belt would have 4 segments).

I would keep them as linkable, it's not significantly less realistic than how linking any other belt can't exceed the `count` of the magazine class.

https://cdn11.bigcommerce.com/s-817fb/images/stencil/500x659/products/5331/69865/IMG_5654__11640.1694970702.jpg